oioioi/programs/models.py

```diff
@@ -326,6 +326,7 @@ class TestReport(models.Model):
     max_score = ScoreField(null=True, blank=True)
     time_used = models.IntegerField(blank=True)
     output_file = FileField(upload_to=make_output_filename, null=True, blank=True)
+    result_percentage = models.FloatField(null=True, blank=True)
```

Comment:
Do we want to add some constraint to force the range of result_percentage's values?

Reply:
what about using fraction field: https://django-fractions.readthedocs.io/en/latest/readme.html

Reply:
It's not perfect (only Decimal fractions).